src/test/java/spoon/test/imports/ImportTest.java

```diff
@@ -1252,4 +1257,23 @@ public void testImportWithGenerics() throws IOException {
 
 		assertTrue(output.contains("import spoon.test.imports.testclasses.withgenerics.Target;"));
 	}
+
+	@Test
+	public void testEqualsImports() {
+		// contract: two imports of same kind with same reference should be equals
+		final Launcher launcher = new Launcher();
+
+		CtType typeA = launcher.getFactory().Type().get(A.class);
+
+		CtImport importsA1 = launcher.getFactory().createImport(ImportKind.TYPE, typeA.getReference());
+		CtImport importsA2 = launcher.getFactory().createImport(ImportKind.TYPE, typeA.getReference());
+
+		assertEquals(importsA1, importsA2);
+		assertEquals(importsA1.hashCode(), importsA2.hashCode());
```

Comment:
will disappear with rebase

Reply:
Actually no: this one is a new test related to CtImport. It's different from the test on CtReference introduced in the other PR.